docs/block-edit-save.md

```diff
@@ -0,0 +1,107 @@
+# Edit and Save
+
+When registering a block, the `edit` and `save` functions provide the interface for how a block is going to be rendered within the editor, how it will operate and be manipulated, and how it will be saved.
+
+## Edit
+
+The `edit` function describes the structure of your block in the context of the editor. This represents what the editor will render when the block is used.
+
+```js
+// Defining the edit interface
+edit() {
+	return <hr />;
+}
+```
+
+This function can return properties through an object argument.
```

Comment:
I'm not sure what's meant by this sentence. Is this trying to describe the behavior of props? In which case, "this function _receives_ properties" is more accurate than "can return".

Reply:
Do we want to shy away from the "props" terminology?